src/components/tabController2/TabBarItem.tsx

```diff
@@ -117,6 +117,8 @@ export default function TabBarItem({
   const {currentPage} = useContext(TabBarContext);
   const itemRef = useRef();
   const itemWidth = useRef(props.width);
+  const sharedLabelStyle = useSharedValue(JSON.parse(JSON.stringify(labelStyle)));
+  const sharedSelectedLabelStyle = useSharedValue(JSON.parse(JSON.stringify(selectedLabelStyle)));
```

Comment:
Looks like this can be solved using only:
```
const sharedLabelStyle = useSharedValue(labelStyle);
const sharedSelectedLabelStyle = useSharedValue(selectedLabelStyle);
```
and in `animatedLabelStyle` returning:
```
return (isActive ? sharedSelectedLabelStyle.value : sharedLabelStyle.value) || {};
```

Reply:
The issue this solves wasn't exposed cause some code is commented out in the example screen 
I pushed a code the expose the bug -> https://github.com/wix/react-native-ui-lib/pull/1325/commits/ff8c46179477d3819825c59bedad972ea79eb1ec